pyaedt/modules/SetupTemplates.py

```diff
@@ -1,2380 +1,1500 @@
-        self.props["SweepRanges"]["Subrange"].append(sweep_range)
-        return self.update()
-
-    @pyaedt_function_handler()
-    def create(self):
-        """Create a sweep.
-
-        Returns
-        -------
-        bool
-            ``True`` when successful, ``False`` when failed.
-
-        """
-        self.oanalysis.InsertSweep(self.setupname, self._get_args())
-        return True
-
-    @pyaedt_function_handler()
-    def update(self):
-        """Update the sweep.
-
-        Returns
-        -------
-        bool
-            ``True`` when successful, ``False`` when failed.
-
-        """
-        self.oanalysis.EditSweep(self.setupname, self.name, self._get_args())
-
-        return True
-
-    @pyaedt_function_handler()
-    def _get_args(self, props=None):
-        """Retrieve properties.
-
-        Parameters
-        ----------
-        props : dict
-             Dictionary of the properties. The default is ``None``, in which case
-             the default properties are retrieved.
-
-        Returns
-        -------
-        dict
-            Dictionary of the properties.
-
-        """
-        if props is None:
-            props = self.props
-        arg = ["NAME:" + self.name]
-        _dict2arg(props, arg)
-        return arg
-
-
-class SetupKeys(object):
-    """Provides setup keys."""
-
-    SetupTemplates = {
-        0: HFSSDrivenAuto,
-        1: HFSSDrivenDefault,
-        2: HFSSEigen,
-        3: HFSSTransient,
-        4: HFSSSBR,
-        5: MaxwellTransient,
-        6: Magnetostatic,
-        7: EddyCurrent,
-        8: Electrostatic,
-        9: Electrostatic,
-        10: ElectricTransient,
-        11: SteadyTemperatureAndFlow,
-        12: SteadyTemperatureOnly,
-        13: SteadyFlowOnly,
-        14: Matrix,
-        15: NexximLNA,
-        16: NexximDC,
-        17: NexximTransient,
-        18: NexximQuickEye,
-        19: NexximVerifEye,
-        20: NexximAMI,
-        21: NexximOscillatorRSF,
-        22: NexximOscillator1T,
-        23: NexximOscillatorNT,
-        24: NexximHarmonicBalance1T,
-        25: NexximHarmonicBalanceNT,
-        26: NexximSystem,
-        27: NexximTVNoise,
-        28: HSPICE,
-        29: HFSS3DLayout,
-        30: Open,
-        31: Close,
-        32: MechTerm,
-        33: MechModal,
-        34: GRM,
-        35: TR,
-        36: TransientTemperatureAndFlow,
-        37: TransientTemperatureOnly,
-        38: TransientFlowOnly,
-        39: MechStructural,
-        40: SiwaveDC3DLayout,
-        41: SiwaveAC3DLayout,
-        42: LNA3DLayout,
-        43: DFIG,
-        44: TPIM,
-        45: SPIM,
-        46: TPSM,
-        47: BLDC,
-        48: ASSM,
-        49: PMDC,
-        50: SRM,
-        51: LSSM,
-        52: UNIM,
-        53: DCM,
-        54: CPSM,
-        55: NSSM,
+LSSM = copy.deepcopy(RmxprtDefault)
+LSSM["WindingConnection"] = False
+
+"""RMxprt LSSM Line-start Synchronous Machine setup properties."""
```

Comment:
It won't let me copy and edit the line. Edit to "RMxprt (Line-Start Synchronous Machine) setup properties."